docs/requirement-set-tables/outlook-1_8.md

```diff
@@ -0,0 +1,69 @@
+| Class | Fields | Description |
+|:---|:---|:---|
+|[AppointmentCompose](/javascript/api/outlook/outlook.appointmentcompose)|[addFileAttachmentFromBase64Async(base64File: string, attachmentName: string, options?: CommonAPI.AsyncContextOptions &  { isInline: boolean }, callback?: (asyncResult: CommonAPI.AsyncResult<string>) => void)](/javascript/api/outlook/outlook.appointmentcompose#addfileattachmentfrombase64async-base64file--attachmentname--options--isinline--callback--asyncresult-)|Adds a file to a message or appointment as an attachment.|
```

Comment:
Seems like we should clean up CommonAPI refs

Reply:
Good catch. I'll add that logic to the post-processor.